docs/scripts/generateProptypes.ts

```diff
@@ -43,6 +43,12 @@ async function generateProptypes(project: XTypeScriptProject, sourceFile: string
       if (propsToNotResolve.includes(name)) {
         return false;
       }
+      if (project.name.includes('x-charts') && sourceFile.includes('AxisTooltip')) {
```

Comment:
Could we also include `ChartsContainer` here?
Is there a reason not to skip those props on all components or at least all `x-charts` components?

Reply:
But this prop-type is so long :sob:

Reply:
What's the issue of too long proptypes?

Reply:
Bundle size in dev mostly (I hope everybody is removing them from the prod bundle).

Reply:
I can reassure you, MUI is automatically removing proptypes in the build thanks to a [babel plugin](https://github.com/oliviertassinari/babel-plugin-transform-react-remove-prop-types) made by a nice contributor 😁

Here is the result if you look in the built package

```js
process.env.NODE_ENV !== "production" ? BarElement.propTypes = {
  // ----------------------------- Warning --------------------------------
  // | These PropTypes are generated from the TypeScript type definitions |
  // | To update them edit the TypeScript types and run "yarn proptypes"  |
  // ----------------------------------------------------------------------
  classes: _propTypes.default.object,
  dataIndex: _propTypes.default.number.isRequired,
  highlightScope: _propTypes.default.shape({
    faded: _propTypes.default.oneOf(['global', 'none', 'series']),
    highlighted: _propTypes.default.oneOf(['item', 'none', 'series'])
  }),
  /**
   * The props used for each component slot.
   * @default {}
   */
  slotProps: _propTypes.default.object,
  /**
   * Overridable component slots.
   * @default {}
   */
  slots: _propTypes.default.object
} : void 0;
```

Reply:
I moved with the remove all big proptypes. Will put them back if it solves some issues asked by users

Reply:
OK this plugin is awesome :star_struck: